tests/stubtest_third_party.py

```diff
@@ -105,6 +106,10 @@ def run_stubtest(dist: Path) -> None:
             print(f"stubtest succeeded for {dist.name}", file=sys.stderr)
 
 
+def has_py3_stubs(dist: Path) -> bool:
+    return len(glob(f"{dist}/*.pyi")) > 0 or len(glob("{dist}/[!@]*/*.pyi")) > 0
```

Comment:
```suggestion
    return len(glob(f"{dist}/*.pyi")) > 0 or len(glob(f"{dist}/[!@]*/*.pyi")) > 0
```

If the dist contains `.pyi` files directly, it has Python 3 stubs, since otherwise they would be all in `@python2`. If it has no `.pyi` files directly, we also consider the second glob. It looks for files in subdirectories not starting with `@`, but why?

(Technically this would break if there was `*` in a directory name, which is possible but won't happen in practice.)

Reply:
The first case is for module distributions (`foo.pyi`), the second for package distributions (`foo/__init__.pyi`, the more common case).

Reply:
For packages, could we just look for `__init__.pyi` specifically, since it's always there? I think it would more obviously say "this is a package".